python/llm/example/GPU/Deepspeed-AutoTP/README.md

```diff
@@ -3,7 +3,9 @@
 This example demonstrates how to run BigDL-LLM optimized low-bit model on multiple [Intel GPUs](../README.md) by leveraging DeepSpeed AutoTP.
 
 ## Requirements
-To run this example with BigDL-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information. For this particular example, you will need at least two GPUs on your machine.
```

Comment:
Remove the requirements and system support parts of this page?
![image](https://github.com/intel-analytics/BigDL/assets/21994046/7ca4c22c-5e40-494e-91b8-386b28f772fa)